Fix Inview

aut0poietic opened the pull request:
Fixed issues with inview that prevented the component from working with adapt_framework 2.4. This update resolves issue #1. 

@moloko you feel like doing a review?

aut0poietic committed:
Fix Inview

Fixed issues with inview that prevented the component from working with adapt_framework 2.4.

moloko commented on js/adapt-lottie.js:
```diff
@@ -44,38 +44,44 @@ define([
             this.$el.imageready(_.bind(function () {
                 this.setReadyStatus();
                 this.animationInit();
+                this.$( '.lottie-container' ).on( 'inview', _.bind( this.inView, this ) );
```
```suggestion
                this.$( '.lottie-container' ).on('inview', this.inView.bind(this));
```

aut0poietic replied:
Thanks @moloko. Done and done.

moloko reviewed (commented):
You should probably amend `"homepage"` and `"issues"` in bower.json 

minor suggestion to use native `Function.bind` instead of `_.bind`

otherwise looks good

aut0poietic committed:
remove _.bind()

Removing _.bind() calls and replacing with .bind(). Also updated the bower.json to point to my project rather than dancgray's. Shame about that.
